.snippets/code/builders/get-started/networks/moonbeam-dev/terminal/compile.md

```diff
@@ -0,0 +1,23 @@
+<div id="termynal" data-termynal>
+  <span data-ty>Compiling try-runtime-cli v0.9.0 (https://github.com/paritytech/substrate?branch=rococo-
+    v1#401c24e8)
+    <br> Compiling frame-benchmarking-cli v3.0.0 (https://github.com/paritytech/substrate?branch=
+    rococo-v1#401c24e8)
+    <br> Compiling cumulus-client-cli v0.1.0 (https://github.com/paritytech/cumulus?branch=rococo
+    -v1#9d89ed65)
+    <br> Compiling moonbeam-rpc-txpool v0.6.0 (/home/purestake/moonbeam/client/rpc/txpool)
+    <br> Compiling moonbeam-rpc-debug v0.1.0 (/home/purestake/moonbeam/client/rpc/debug)
+    <br> Compiling moonbeam-rpc-trace v0.6.0 (/home/purestake/moonbeam/client/rpc/trace)
+    <br> Compiling cumulus-client-network v0.1.0 (https://github.com/paritytech/cumulus?branch=ro
+    coco-v1#9d89ed65)
+    <br> Compiling cumulus-client-consensus-relay-chain v0.1.0 (https://github.com/paritytech/cum
+    ulus?branch=rococo-v1#9d89ed65)
+    <br> Compiling polkadot-test-service v0.8.29 (https://github.com/paritytech/polkadot?branch=r
+    ococo-v1#b64741e6)
+    <br> Compiling cumulus-client-collator v0.1.0 (https://github.com/paritytech/cumulus?branch=r
+    ococo-v1#9d89ed65)
+    <br> Compiling cumulus-client-service v0.1.0 (https://github.com/paritytech/cumulus?branch=ro
+    coco-v1#9d89ed65)
```

Comment:
Can you remove the new lines here between ro and coco? 
<img width="710" alt="Screenshot 2024-01-10 at 8 10 10 PM" src="https://github.com/moonbeam-foundation/moonbeam-docs/assets/5834262/900d7f4f-c3da-4361-ba90-803ef099070f">

Reply:
good catch! thank you!